web/client-api/src/main/java/io/deephaven/web/client/api/tree/enums/JsAggregationOperation.java

```diff
@@ -88,7 +88,6 @@ public class JsAggregationOperation {
      */
     public static final String SKIP = "Skip";
 
-    @JsIgnore
```

Comment:
Since TS doesn't really support proper enum types, I've tried to avoid having an actual enum class in our JS API, which this would sort of change (plus it is awkward/weird to say "well its a function for enums, but we just don't bother to actually put it on the enum instance, since enums aren't objects").

Can you elaborate a bit on how you want to use this? My hunch is you're going to loop through all known not-enum values - would it make more sense to expose a "please tell me all the values", "please tell me all the values _that match this `columnType`" etc set of methods?

Reply:
We need to be able to feature-detect whether the API supports the SKIP aggregation operation. @mofojed suggested using this method for the check.